assets/rpvoicechat/lang/de.json

```diff
@@ -27,10 +27,23 @@
     "Command.ForceNameTags.Help": "Wenn die Namensschilder immer eingeschaltet sind, bewirkt diese Einstellung nichts. Der Standardwert ist <strong>true</strong>",
     "Command.ForceNameTags.Success.Enabled": "Namensschilder werden während des Sprechens wiedergegeben, auch wenn sie versteckt waren.",
     "Command.ForceNameTags.Success.Disabled": "Namensschilder werden während des Sprechens NICHT wiedergegeben, wenn sie versteckt wurden.",
+    "Command.EncodeAudio.Desc": "Schaltet die Audiokodierung um. Siehe Hilfe für mehr Informationen",
+    "Command.EncodeAudio.Help": "Die Audiocodierung verringert die Bandbreitennutzung drastisch, beeinträchtigt aber die Audioqualität und kann Artefakte erzeugen. Wenn die Bandbreite für Sie kein Problem darstellt oder Sie Probleme mit der Audioqualität haben - versuchen, die Kodierung zu deaktivieren. Der Standardwert ist <strong>on</strong>",
+    "Command.EncodeAudio.Success.Enabled": "Audiocodierung ist aktiviert",
+    "Command.EncodeAudio.Success.Disabled": "Audiocodierung ist deaktiviert",
     "Hotkey.ModMenu": "RPVoice: Mod Menü",
     "Hotkey.VoiceLevel": "RPVoice: Sprachdistanz ändern",
     "Hotkey.PTT": "RPVoice: Push to talk",
     "Hotkey.Mute": "RPVoice: Stumm schalten",
+    "Gui.AudioWizardDialog.TitleBar": "Audio-Assistent",
+    "Gui.AudioWizardDialog.FirstParagraph": "Willkommen beim Audiokonfigurationsassistenten, der Ihnen helfen wird, Ihr Eingabegerät zu konfigurieren. Sie sollten sich jetzt hören können. Wenn das nicht der Fall ist, stellen Sie sicher, dass dies Ihr Gerät ist:",
+    "Gui.AudioWizardDialog.SecondParagraph": "Konfigurieren Sie nun Ihre Aufnahmelautstärke. Drücken Sie die Taste unten und beginnen Sie wie gewohnt zu sprechen. Sprechen Sie so lange, bis der Fortschrittsbalken gefüllt ist. Andere Spieler können Sie im Moment nicht hören, also machen Sie sich keine Sorgen um sie.",
+    "Gui.AudioWizardDialog.Done": "Das war's! Die Lautstärke Ihrer Aufnahme wurde automatisch kalibriert, viel Spaß beim Spielen.",
+    "Gui.Button.NotNow": "Nicht jetzt",
+    "Gui.Button.Sure": "Okay",
+    "Gui.FirstLaunchDialog.TitleBar": "Willkommen beim RPVoiceChat!",
+    "Gui.FirstLaunchDialog.FirstParagraph": "Es sieht so aus, als ob Sie den RPVoiceChat mod zum ersten Mal benutzen. Möchten Sie Ihr Eingabegerät automatisch konfigurieren lassen? Es wird weniger als eine Minute dauern.",
+    "Gui.FirstLaunchDialog.SecondParagraph": "Dieses Fenster wird nur einmal angezeigt, aber Sie können den Konfigurationsassistenten jederzeit über das Mod-Menü (\"{0}\"-Taste) aufrufen.",
```

Comment:
Does this say "from the mod menu" or "using the mod menu"? Given that this is first time player uses the mod they may think that pressing P would be enough to open audio wizard and may get a bit lost as to how to proceed

Reply:
It says, "You can call up the configuration wizard at any time via the Mod menu (p)."

Reply:
Sounds a bit ambiguous but I think it will be fine 👍